### ISSUE TITLE
Question about encoding image 

### ISSUE THREAD

**Issue by Eric07110904** [eladrich/pixel2style2pixel#284]
Thanks for your awesome works, I have a question about GAN inversion.
I used the psp to do GAN inversion in anime domain(512x512 300k images), and used pre-trained anime StyleGAN2(512x512 ).
![image](https://user-images.githubusercontent.com/42273490/187056965-3613789d-4f1d-4f7b-a5ed-086d58b0eb56.png)
After training 100,000 iteration with batch_size=4, I observed two problems. 
1. detaied structure of anime face(it seems that my model didn't capture **the part of mouth、wink**)
![image](https://user-images.githubusercontent.com/42273490/187056609-677ec0cb-a007-4c3a-b236-340dc8c2f1e5.png)
![image](https://user-images.githubusercontent.com/42273490/187059643-881c14c0-3747-4051-8db7-4affba414839.png)

3. output is blured
![image](https://user-images.githubusercontent.com/42273490/187056592-feea3ec3-8e00-421f-ba18-cb464a662f9a.png)

Do you have any suggentstion about solving two problems? 
I am wondering if any parameter is set wrong or should i **do more iterations** or add **w_norm loss**?
Thanks for your reply!

**Comment by yuval-alaluf** [comment]
Using the ID loss is a bit strange here since it was trained for real face images and your anime dataset is out of domain for this. 
Other than that, it could be that pSp is not able to fully capture all the details here. You could try other more advanced encoders such as ReStyle and Hyperstyle, or try optimization-based approaches like PTI.
